packages/contracts/config/.env.example

```diff
@@ -11,6 +11,7 @@
 
 # REQUIRED
 DEPLOY_MNEMONIC='your mnemonic here'
+AGGREGATOR_ADDRESS='0x17FC6B6c35f4C4A1e92D9d878aC001EBbF1d8e2D'
```

Comment:
Might want to specify whether these are required to match.  I'm assuming the AGGREGATOR_ADDRESS is the pubkey to the corresponding aggregator mnemonic, not the deploy mnemonic, but could be confusing to those trying to spin up.

Reply:
Yep, your assumption is right, but I'll clarify